scripts/machine_translation/README.md

```diff
@@ -30,6 +30,7 @@ python3 train_transformer.py \
     --save_dir transformer_base_wmt2014_en_de_${SUBWORD_ALGO} \
     --cfg transformer_base \
     --lr 0.002 \
+    --num_accumulated 32 \
```

Comment:
hmm was this missed from before?

Reply:
It's the default argument. Just add it here to make it more explicit.